.github/workflows/go-check.yml

```diff
@@ -8,26 +8,20 @@ jobs:
   unit:
     runs-on: ubuntu-latest
     name: All
-    env:
-      RUNGOGENERATE: false
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
+      - id: config
+        uses: protocol/.github/.github/actions/read-config@master
       - uses: actions/setup-go@v3
         with:
-          go-version: "1.19.x"
+          go-version: 1.20.x
```

Comment:
aha! I thought 1.19.x was no longer the latest. I'll go ahead and put back in all my atomic.Uint's